src/tag/tag.tsx

```diff
@@ -18,9 +19,13 @@ export const Tag = ({
             $interactive={interactive}
             {...otherProps}
         >
-            {iconPosition === "left" && icon}
-            <Label>{children}</Label>
-            {iconPosition === "right" && icon}
+            {iconPosition === "left" &&
+                icon &&
+                React.cloneElement(icon, { "aria-hidden": "true" })}
```

Comment:
we can use boolean directly here, React will handle the casting to the right html attribute

can the other instance be updated as well?

```suggestion
                React.cloneElement(icon, { "aria-hidden": true })}
```